libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -141,12 +168,73 @@ static void _destroyMqttConnection( _mqttConnection_t * pMqttConnection );
  */
 static IotMqttError_t _subscriptionCommon( IotMqttOperationType_t operation,
                                            IotMqttConnection_t mqttConnection,
+                                           IotMqttSerializeSubscribe_t serializeSubscription,
                                            const IotMqttSubscription_t * pSubscriptionList,
                                            size_t subscriptionCount,
                                            uint32_t flags,
                                            const IotMqttCallbackInfo_t * pCallbackInfo,
                                            IotMqttOperation_t * const pOperationReference );
 
+/**
+ * @cond DOXYGEN_IGNORE
+ * Doxygen should ignore this section.
+ *
+ * Declaration of local MQTT serializer override selectors
+ */
+#if IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES == 1
+_IOT_MQTT_SERIALIZER_OVERRIDE_SELECTOR(
```

Comment:
Nit: Rename to `_SERIALIZER_OVERRIDE_SELECTOR` (internal macros do not need the library name)